pkg/types/baremetal/validation/platform.go

```diff
@@ -24,8 +24,12 @@ func ValidatePlatform(p *baremetal.Platform, n *types.Networking, fldPath *field
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("libvirtURI"), p.LibvirtURI, err.Error()))
 	}
 
-	if err := validate.URI(p.IronicURI); err != nil {
-		allErrs = append(allErrs, field.Invalid(fldPath.Child("ironicURI"), p.LibvirtURI, err.Error()))
+	if err := validate.IP(p.ClusterProvisioningIP); err != nil {
```

Comment:
validate which range this can belong to. machine/pod/service..

Reply:
Ack, it needs to be a dedicated additional network so it must not overlap

Reply:
I raised https://github.com/openshift/installer/issues/2210 so this validation can perhaps be a follow-up PR?